sdk/core/azure-core/src/http/curl/curl_connection_pool_private.hpp

```diff
@@ -41,6 +44,10 @@ namespace Azure { namespace Core { namespace Http {
     // Give access to private to this tests class
     friend class Azure::Core::Test::CurlConnectionPool_connectionPoolTest_Test;
 #endif
+  private:
+    // The cttor and dttor of this member makes sure of calling the libcurl global init and cleanup
+    AZ_CORE_DLLEXPORT static CurlGlobalStateForAzureSdk CurlGlobalState;
```

Comment:
Given this is a static we won't call curl_global_cleanup until the process exits.  I don't see any issue there.  If that is the intention, Add a comment around this line to note the implicit behavior we have.

Reply:
Yes, I added documentation about it. If the application exits or if the SDK is unloaded (dll)

Reply:
I think this should be in the Curl Transport code so that our SDK has no hard dependency on Curl.